src/InvoiceGenerator/InvoiceGenerator.BusinessLogic/ValidationBLogic.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Mail;
+
+namespace InvoiceGenerator.BusinessLogic
+{
+    public class ValidationBLogic
+    {
+        public string[] validateUserDetails(string clientName, string clientAddress, string contactName, string contactEmail) // Validate all user inputs on client details entry screen
```

Comment:
Consider taking in the ClientViewModel here, as it will make the arguments list smaller.